tests/cross_validation/c_harness/make_figs.py

```
@@ -0,0 +1,294 @@
+        c2, s2 = math.cos(2 * psi), math.sin(2 * psi)
+        return a * c2 + b * s2, b * c2 - a * s2  # F+, Fx (sinZeta=1 for H1)
+
+    # write CSV for the C harness and run it
+    with open("sweep_params.csv", "w") as f:
+        for r in sets:
+            f.write(" ".join(f"{r[k]:.17g}" if k != "mode" else f"{r['mode']}"
+                             for k in ["mode", "alpha", "delta", "f0", "f1", "f2",
+                                       "phi0", "psi", "aplus", "across", "asini",
+                                       "ecc", "period", "argp", "tp", "fhet"]) + "\n")
+    print("running compiled-LAL sweep harness ...", flush=True)
+    subprocess.run(["./harness_sweep", earth, sun, "sweep_params.csv",
```

Ruff (0.15.17)

[error] `subprocess` call: check for execution of untrusted input

(S603)